src/@types/global.d.ts

```diff
@@ -115,6 +115,8 @@ declare global {
         mxAutoRageshakeStore?: AutoRageshakeStore;
     }
 
+    interface Electron { }
```

Comment:
IMO the def above should be `electron?: unknown;` that way we can't use any fields on it, an empty interface can still be keyed